packages/openactive-integration-tests/test/features/details-capture/customer-details-capture-identifier/implemented/customer-identifier-capture-test.js

```diff
@@ -0,0 +1,81 @@
+/* eslint-disable no-unused-vars */
+const chai = require('chai');
+const chakram = require('chakram');
+const { FeatureHelper } = require('../../../../helpers/feature-helper');
+const { GetMatch, C1, C2, B } = require('../../../../shared-behaviours');
+const { FlowHelper } = require('../../../../helpers/flow-helper');
+const { RequestState } = require('../../../../helpers/request-state');
+
+/* eslint-enable no-unused-vars */
+
+FeatureHelper.describeFeature(module, {
+  testCategory: 'details-capture',
+  testFeature: 'customer-details-capture-identifier',
+  testFeatureImplemented: true,
+  testIdentifier: 'customer-identifier-capture',
+  testName: 'C1, C2 and B - identifier is reflected back',
+  testDescription: 'Identifier from the Customer supplied by Broker should be reflected back by booking system.',
+  // The primary opportunity criteria to use for the primary OrderItem under test
+  testOpportunityCriteria: 'TestOpportunityBookablePaid',
```

Comment:
Should use TestOpportunityBookable as does not depend on the opportunity being a non-free session

Reply:
Understood